deploy/charts/harvester/values.yaml

```diff
@@ -93,6 +93,25 @@ kubevirt:
       ##
       emulatedMachines: ["q35","pc-q35*","pc","pc-i440fx*"]
 
+    customizeComponents:
+      patches:
+        - patch: '{"webhooks":[{"name":"kubevirt-validator.kubevirt.io","failurePolicy":"Ignore"},{"name":"kubevirt-update-validator.kubevirt.io","failurePolicy":"Ignore"}]}'
+          resourceName: virt-operator-validator
+          resourceType: ValidatingWebhookConfiguration
+          type: strategic
+        - patch: '{"spec":{"template":{"spec":{"containers":[{"name":"virt-api", "resources":{"limits":{"cpu":"8","memory":"1Gi"}}}]}}}}'
+          resourceName: virt-api
+          resourceType: Deployment
+          type: strategic
+        - patch: '{"spec":{"template":{"spec":{"containers":[{"name":"virt-controller", "resources":{"limits":{"cpu":"8","memory":"1Gi"}}}]}}}}'
+          resourceName: virt-controller
+          resourceType: Deployment
+          type: strategic
+        - patch: '{"spec":{"template":{"spec":{"containers":[{"name":"virt-handler", "resources":{"limits":{"cpu":"8","memory":"1Gi"}}}]}}}}'
+          resourceName: virt-handler
+          resourceType: DaemonSet
+          type: strategic
```

Comment:
The CPU limit `8` has increased dramatically in contrast with the requested CPU `10m`; any reasons why it will consume such high CPU usage?
btw, any related issue of why the upstream did not set the resource limit of those components? prefer to leave a comment for the future track if it has anyone.

Reply:
> The CPU limit 8 has increased dramatically in contrast with the requested CPU e.g., 10m; any reasons why it will consume such high CPU usage?

In US devices, I have noticed that the CPU sometimes goes above 8C, more detailed analysis will be done later.


> btw, any related issue of why the upstream did not set the resource limit of those components?  prefer to leave a comment for the future track if it has anyone.

I did not find an explanation for the upstream, but I created an issue to track.  https://github.com/kubevirt/kubevirt/issues/8485